client/src/components/common/Product/index.tsx

```diff
@@ -28,11 +29,14 @@ export const Product: React.FC<ProductProps> = (props) => {
   return (
     <li className="product">
       <div className="image-wrapper">
-        {srcLoading ? (
-          <img src={imageUrl} loading={imgLoading} alt="no" />
-        ) : (
-          <div className="like-img"></div>
-        )}
+        <Link to={`/detail/${id}`}>
+          {eagerLoading ? (
+            <img src={imageUrl} loading="eager" alt="no" />
+          ) : (
+            <img src={imageUrl} loading="lazy" alt="no" />
+          )}
+        </Link>
```

Comment:
```suggestion
        <Link to={`/detail/${id}`}>
           {srcLoading ? (
            <img src={imageUrl} loading={imgLoading} alt="no" />
            ) : (
              <div className="like-img"></div>
            )}
        </Link>
```

기존 dev 내용과 합치시면 될 것 같아요!
지금 dev 있는 내용이 덮어씌워진 것 같습니다 ㅠ